Tests/UnitTests/Examples/Algorithms/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory_if(Alignment ACTS_BUILD_ALIGNMENT)
 add_subdirectory(Digitization)
+add_subdirectory(HoughTransform)
```

Comment:
Maybe the folder name should be `Seeding`, so that it represents the category or group, rather than the individual test.